src/libs/ReportActionsUtils.js

```diff
@@ -547,6 +547,17 @@ function getIOUReportIDFromReportActionPreview(reportAction) {
     return lodashGet(reportAction, 'originalMessage.linkedReportID', '');
 }
 
+/**
+ * Find the reportAction whose childReportID is childReportID in reportActions with reportID
+ *
+ * @param {String} childReportID
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getParentReportActionInReport(childReportID, reportID) {
```

Comment:
NAB but let's move this function to be below `getParentReportAction`

Reply:
okay